floris/simulation/wake_deflection/base_velocity_deflection.py

```diff
@@ -124,40 +124,49 @@ def calculate_effective_yaw_angle(self, x_locations, y_locations,
             HH = turbine.hub_height
             aI = turbine.aI
             TSR = turbine.tsr
-
             V = flow_field.v
             W = flow_field.w
+            Uinf = np.mean(flow_field.wind_map.grid_wind_speed)
+
+            eps = self.eps_gain * D  # Use set value
+            xLocs = x_locations - coord.x1
+            idx = np.where((np.abs(xLocs) < D / 4))
 
-            yLocs = y_locations - coord.x2
-            zLocs = z_locations - (HH)
+            yLocs = y_locations[idx] + 0.01 - coord.x2
```

Comment:
@jrannoni Wondering what purpose the offsets have? Is it avoiding some error? And if so, should we check for that error?